pylint/lint/pylinter.py

```diff
@@ -1567,7 +1567,7 @@ def _add_one_message(
             self.stats.by_msg[message_definition.symbol] = 1
         # Interpolate arguments into message string
         msg = message_definition.msg
-        if args:
+        if args is not None:
```

Comment:
@DanielNoord RE this [comment](https://github.com/PyCQA/pylint/pull/6015#discussion_r841270789):

This is a classic Falsey bug. When a function takes `None` as it's default arg then it's always best to compare the argument to `None` otherwise you risk catching other Falsey objects. In this case it was unintentionally catching the empty string I passed 😄

Reply:
I lost more than two days of work to 0 and 0.0 being falsey in the last year, I send my heartfelt condolences